toolkit/mozapps/handling/ContentDispatchChooser.jsm

```diff
@@ -227,6 +227,23 @@ let ContentDispatchChooserTelemetry = {
   },
 };
 
+const replaySchemeMap = {
+  library: 'https://replay.io/view'
+};
+
+function mayRedirectToReplayBrowser (aURI, aPrincipal, aBrowsingContext) {
+  if (aURI.scheme.toLowerCase() === 'replay') {
+    const newUrl = replaySchemeMap[aURI.filePath];
+    
+    if (newUrl) {
+      const tabBrowser = aBrowsingContext.topFrameElement.getTabBrowser();
+      tabBrowser.loadURI(newUrl, { triggeringPrincipal: aPrincipal });
+    
+      return true;
+    }
+  }
+}
+
 class nsContentDispatchChooser {
   /**
    * Prompt the user to open an external application.
```

Comment:
The prompts here make it sound like this is runs when you open a URL from within the Replay browser, are we sure this also runs if something tries to open a replay link from due to `CFBundleURLSchemes`?

Reply:
I've been testing by typing `replay:library` in chrome which prompts to "Open in Replay" and then launches Replay